sdk/monitor/opentelemetry-exporter-azure-monitor/src/utils/connectionStringParser.ts

```diff
@@ -69,4 +72,17 @@ export class ConnectionStringParser {
 
     return result;
   }
+
+  public static sanitizeUrl(url: string) {
+    let newUrl = url.trim();
+    if (newUrl.indexOf("https") < 0) {
+      // Try to update http to https
+      newUrl = newUrl.replace("http", "https");
+    }
+    // Remove final slash if present
+    if (newUrl.slice(-1) == "/") {
```

Comment:
NIT: I think this results in an unneeded array (it's small, but you're really just checking the last character of an array you already have - would `newUrl[newUrl.length - 1]` suffice?

Or is this needed because of some other constraint?